lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -114,24 +114,27 @@ <h3 class="hd hd-3">${_("Regenerate Certificates")}</h3>
             <p class="under-heading">
                 ${_('To regenerate certificates for your course, choose the learners who will receive regenerated certificates and click Regenerate Certificates.')}
             </p>
-
-                <label style="display: inline" for="certificate_status_${section_data['status'].downloadable}">
-                    <input id="certificate_status_${section_data['status'].downloadable}" type="checkbox" name="certificate_statuses" value="${section_data['status'].downloadable}">
-                    ${_("Regenerate for learners who have already received certificates. ({count})").format(count=section_data['certificate_statuses_with_count'].get(section_data['status'].downloadable, 0))}
-                </label>
-                <br/>
-                <label style="display: inline" for="certificate_status_${section_data['status'].notpassing}">
-                    <input id="certificate_status_${section_data['status'].notpassing}" type="checkbox" name="certificate_statuses" value="${section_data['status'].notpassing}">
-                    ${_("Regenerate for learners who have not received certificates. ({count})").format(count=section_data['certificate_statuses_with_count'].get(section_data['status'].notpassing, 0))}
-                </label>
-                <br/>
-                <label style="display: inline" for="certificate_status_${section_data['status'].error}">
-                    <input id="certificate_status_${section_data['status'].error}" type="checkbox" name="certificate_statuses" value="${section_data['status'].error}">
-                    ${_("Regenerate for learners in an error state. ({count})").format(count=section_data['certificate_statuses_with_count'].get(section_data['status'].error, 0))}
-                </label>
-                <br/>
-                <br/>
-
+            <fieldset>
+                <legend class="sr">${_('Choose learner types for regeneration')}</legend>
```

Comment:
There wasn't a legend before so I included one for correct-ness, but made it invisible. There is sufficient context in the heading and paragraph that precede it. This keeps it visually the way it has been.